docs/farming-&-staking/staking/intro.md

```diff
@@ -63,29 +63,35 @@ Our suggested specs are not necessarily applicable to Stake Wars. We encourage a
 **Network**:
 - The minimum symmetric networking speed is set to 500 Mbit/s.
 
+### Folder structure
 
+Starting with Gemini 3h, **Subspace Node** will create the following directory structure:
+
+- subspace-node/db
+- subspace-node/domains/0/keystore
+- subspace-node/domains/0/db
+- subspace-node/domains/1/keystore
+- subspace-node/domains/1/db
+- subspace-node/network
+
+For operators it means that you can no longer create a keystore in a random location, the keystore has to be located in a specific subfolder of the desired domain and will be
```

Comment:
Are we missing the end of the sentence here?

Reply:
Hm, not sure what happened here. 
I don't like the phrasing though, so just rewrite the whole sentence.